modules/space/rendering/renderablesatellites.cpp

```diff
@@ -76,197 +75,16 @@ namespace {
         "Trail Fade",
         "This value determines how fast the trail fades and is an appearance property. "
     };
-    
-    constexpr const char* KeyFile = "Path";
-    constexpr const char* KeyLineNum = "LineNumber";
+    static const openspace::properties::Property::PropertyInfo UpperLimitInfo = {
```

Comment:
is this needed since it's in the parent class?
 same for trail fade?

Reply:
All of the PropertyInfo declarations are actually not in the parent class. Throughout OpenSpace these are declared in a separate namespace rather than in the class, so I just stayed with that convention. The downside of this is that these are repeated in base classes.

However, I deleted this particular `upperLimitInfo` variable just now because I had already removed the upper limit from satellite (only smallbody uses it because we don't have files with huge numbers of satellites) but forgot to remove it here.